Tools/autotest/sim_vehicle.py

```diff
@@ -678,6 +676,10 @@ def start_vehicle(binary, opts, stuff, spawns=None):
         c = ["-I" + str(i)]
         if spawns is not None:
             c.extend(["--home", spawns[i]])
+        if opts.sysid is None:
```

Comment:
this looks like it will always pass a sysid to ardupilot. That would override any sysid set with a parameter by a user

Reply:
Yea, I wonder if having an explicit option for automatically setting the sysid would be best. I have another branch for that here: https://github.com/araneg/ardupilot/tree/sim_vehicle_sysid_alt

Let me know, and I'll force push that branch onto this PR.